index.html

```diff
@@ -19,11 +21,13 @@
     </div> -->
     <hr>
     <input type="text" id="operator">
+    <!-- варианты операторов,которые вводит пользователь -->
     <hr>
-    <input id="number2">
+    <input id="number2" type="text" onkeyup="return control(this);">
```

Comment:
Если твоя цель сделать провалидировать вводимые денные, то есть допустить ввод только цифр: type="number"
```suggestion
    <input id="number2" type="number">
```